core/templates/components/button-directives/exploration-embed-button-modal.component.spec.ts

```diff
@@ -32,7 +32,7 @@ class MockActiveModal {
 }
 
 class MockSiteAnalyticsService {
-  registerOpenEmbedInfoEvent(id: string): void {
+  registerOpenEmbedInfoEvent(_id: string): void {
```

Comment:
I didn't get this change? What is happening here?

Reply:
We are using underscore for unused variables, this function is going to be used later, there I have to use an underscore before it.

Reply:
Why we are not removing it from here and where it is called?

Reply:
Actually it is not called, here we have just general definition and at other places, this function is defined fully.